ruby/red-arrow/lib/arrow/table.rb

```diff
@@ -492,7 +504,8 @@ def pack
     #   @macro join_common_after
     #
     # @since 7.0.0
-    def join(right, keys, type: :inner, left_outputs: nil, right_outputs: nil)
+    def join(right, keys = nil, type: :inner, left_outputs: nil, right_outputs: nil)
```

Comment:
Could you remove spaces around `=` for default value? We use the style in this project.

```suggestion
    def join(right, keys=nil, type: :inner, left_outputs: nil, right_outputs: nil)
```

Reply:
OK. Thank you.